web/view_history.js

```diff
@@ -12,7 +12,7 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-/* globals PDFJS, Promise */
+/* globals Promise */
```

Comment:
Thanks to https://github.com/mozilla/pdf.js/blob/master/.jshintrc#L7, you should also be able to remove `Promise` here (might apply elsewhere too).